core/src/navigation/TopNav.html

```diff
@@ -163,19 +168,24 @@
       },
       toggleDropdownState(name) {
         const dropDownStates = this.get().dropDownStates || {};
-        dropDownStates[name] = GenericHelpers.getNegatedBoolString(
-          dropDownStates[name]
-        );
+        const dropDownState = GenericHelpers.getNegatedBoolString(dropDownStates[name]);
+        this.closeAllDropdowns();
+        dropDownStates[name] = dropDownState;
         this.set({
           dropDownStates
         });
       },
-      closeDropdown(name) {
+      closeAllDropdowns() {
```

Comment:
I would refactor this method, also in `ContextSwitcher` component, to the following:
```
closeAllDropdowns() {
  const dropDownStates = this.get().dropDownStates || {};
  Object.keys(dropDownStates).forEach(k => { dropDownStates[k] = 'false' });
  this.set({dropDownStates});
}
```

Reply:
`dropDownStates` is an empty object at the beginning. Adding the _foreach_ loop without checking if the array of keys is not empty, would cause errors.